pystac/cache.py

```diff
@@ -20,12 +25,12 @@ def get_cache_key(stac_object):
     if href is not None:
         return (href, True)
     else:
-        ids = []
-        obj = stac_object
+        ids: List[str] = []
+        obj: Optional[ps.STACObject] = stac_object
         while obj is not None:
             ids.append(obj.id)
             obj = obj.get_parent()
-        return ('/'.join(ids), False)
+        return ("/".join(ids), False)
```

Comment:
Looks like you are standardizing on `"`, so a style guide might be in order.  Doesn't have to be much, but I also noticed the 88 char max for lines which is unusual

Reply:
I can thoroughly recommend [Black](https://pypi.org/project/black/) to auto-format Python code. Then there's no need to review these sorts of details anymore.

Reply:
Agreed - and the reason why this changed is that this PR moves off of yapf and starts using Black (mentioned in #316).  This is where the 88 line limit comes from. Definitely worth a mention in the README, but the style guide will be basically "we use Black", which makes it easier on the project and like Victor said, removes the need for formatting reviews.